.gitignore

```diff
@@ -12,3 +12,4 @@ _site
 
 # Bundler vendor folder
 vendor
+.bundle
```

Comment:
Alternatively, you could check in the .bundle folder, and then you don't have to specify `--path vendor/bundle` everywhere.

Reply:
.bundle only gets created when you use `--path`. Otherwise gems are installed to /var/lib/gems/2.3.0/gems.

Reply:
Meh, I just switched to `--deployment` at @jvperrin's suggestion. Since this works I'm gonna merge it.